store/seed/test/10070__db.sql

```diff
@@ -699,6 +699,62 @@ VALUES
         1624558090,
         '2021v1'
     );
+INSERT INTO
+    db (
+        id,
+        creator_id,
+        updater_id,
+        instance_id,
+        project_id,
+        name,
+        character_set,
+        collation,
+        sync_status,
+        last_successful_sync_ts,
+        schema_version
+    )
+VALUES
+    (
+        7025,
+        101,
+        101,
+        6001,
+        3007,
+        'db3_bytebase',
```

Comment:
nit: db3_tenant1, db3_tenant2

"bytebase" has special meanings and "shoes" is too general (nike, adidas would be a better name).